config/brakeman.ignore

```diff
@@ -111,6 +111,29 @@
       ],
       "note": ""
     },
+    {
+      "warning_type": "File Access",
+      "warning_code": 16,
+      "fingerprint": "7501c46c81f7d45a1f7629d9a9321e68c6e4afa3e5465264e7574ef1bfbbf062",
+      "check_name": "FileAccess",
+      "message": "Model attribute used in file name",
+      "file": "app/sidekiq/split_worker.rb",
+      "line": 19,
+      "link": "https://brakemanscanner.org/docs/warning_types/file_access/",
+      "code": "FileUtils.remove_dir(\"#{ExtractionJob.find(extraction_job_id).extraction_folder}/tmp\")",
+      "render_path": null,
+      "location": {
+        "type": "method",
+        "class": "SplitWorker",
+        "method": "perform"
+      },
+      "user_input": "ExtractionJob.find(extraction_job_id).extraction_folder",
+      "confidence": "Medium",
+      "cwe_id": [
+        22
+      ],
+      "note": ""
```

Comment:
Should you put a note in here to say why it isn't a concern?

Reply:
Good idea! 💃